frontend/packages/akt/public/i18n/fi-FI/translation.json

```diff
@@ -243,7 +243,7 @@
           "to": "Mihin"
         },
         "switch": {
-          "canPublish": "Saa julkaista internetissä",
+          "permissionToPublish": "Julkaisulupa",
```

Comment:
Tästä ei ehkä tullut asiakkaalta mitään kommenttia, mutta kiinnitin joskus itse huomiota, että kaikkialla muualla käytössä termi "Julkaisulupa", niin varmaan hyvä ottaa se käyttöön täälläkin.

Reply:
Hämärä muistikuva että tästä olisi tosiaan keskusteltu ja asiakkaaltakin komppausta tullut. Ainakin konsistentimpi näin.